platform/linuxbsd/wayland/display_server_wayland.cpp

```diff
@@ -1449,6 +1450,128 @@ void DisplayServerWayland::window_start_resize(WindowResizeEdge p_edge, WindowID
 	wayland_thread.window_start_resize(p_edge, p_window);
 }
 
+void DisplayServerWayland::_window_update_hdr_state(WindowID p_window_id) {
+	ERR_FAIL_COND(!windows.has(p_window_id));
+	WindowData &wd = windows[p_window_id];
```

Comment:
Nit: Since this is an internal method, you could pass in a reference to the `WindowData` and avoid the second lookup.